src/constants/numbers.ts

```diff
@@ -2,3 +2,5 @@ export const WAD_DECIMALS = 18
 
 export const SECONDS_IN_DAY = 24 * 60 * 60
 export const SECONDS_IN_HOUR = 60 * 60
+
+export const PROJECT_PAY_CHARACTER_LIMIT = 16
```

Comment:
@wraeth-eth given https://github.com/jbx-protocol/juice-interface/pull/1007, would it make sense to increase this slightly? Not sure, your call 🙏

Reply:
Oh for sure - if we have plans to wrap it, lets increase maybe to 32? WDYT?

Reply:
Could even go bigger tbh:
![image](https://user-images.githubusercontent.com/104132113/168406261-fc22f48f-526b-4e7d-bdb8-094a77c49744.png)

Reply:
![image](https://user-images.githubusercontent.com/104132113/168406320-747b7e46-b7ba-4805-b8c5-1eeefef8b782.png)

Reply:
On second thought I think I'm in favour of keeping it at 16. Not sure we want to encourage long buttons, generally not smart from UX pov.

In other words, lgtm!

Reply:
🙌